main.go

```diff
@@ -123,26 +123,27 @@ func ensureFastlaneVersionAndCreateCmdSlice(forceVersion, gemfilePth string) ([]
 
 	log.Printf("Gemfile exist, checking fastlane version from Gemfile.lock")
 
-	gemfileDir := filepath.Dir(gemfilePth)
-	gemfileLockPth := filepath.Join(gemfileDir, "Gemfile.lock")
-
 	bundleInstallCalled := false
-	if exist, err := pathutil.IsPathExists(gemfileLockPth); err != nil {
-		return nil, "", err
-	} else if !exist {
-		log.Printf("Gemfile.lock not exist at: %s, running 'bundle install' ...", gemfileLockPth)
+	gemfileDir := filepath.Dir(gemfilePth)
+	gemfileLockPth, err := gems.GemFileLockPth(gemfileDir)
+	if err != nil {
```

Comment:
PLease update logs to refer to gems lockfile generally, not Gemfile.lock specifically.

Reply:
Updated.